.pre-commit-config.yaml

```diff
@@ -33,3 +31,12 @@ repos:
       - id: check-yaml
         exclude: ^continuous_integration/recipe/
       - id: check-added-large-files
+  - repo: local
+    hooks:
+      - id: cargo-fmt
+        name: cargo fmt
+        description: Format files with cargo fmt.
+        entry: cargo +nightly fmt
+        language: system
+        types: [rust]
+        args: ['--manifest-path', './dask_planner/Cargo.toml', '--verbose', '--']
```

Comment:
I'm not too familiar with the syntax but does this imply users should have the nightly cargo-fmt tool setup already?

Reply:
It does - thanks for the reminder, I will add a blurb to `CONTRIBUTING.md` explaining how to install the nightly toolchain to run the pre-commit hooks properly